src/components/layout/layout.tsx

```diff
@@ -255,39 +256,43 @@ const Layout = ({
             break
         case 'partners':
             Navigation = <NavPartners hide_login_signup={no_login_signup} />
-            FooterNav = <Footer />
+            FooterNav = <LoadableFooter />
             break
         case 'security':
             Navigation = <NavSecurity />
-            FooterNav = <Footer />
+            FooterNav = <LoadableFooter />
             break
         case 'ebook':
             Navigation = <Nav hide_signup_login />
-            FooterNav = <Footer />
+            FooterNav = <LoadableFooter />
             break
         case 'landing-page':
             Navigation = <NavInterim landing_type />
-            FooterNav = <Footer no_footer_links />
+            FooterNav = <LoadableFooter no_footer_links />
             break
         case 'jump-indices':
             Navigation = <NavJumpIndice />
-            FooterNav = <Footer is_ppc={is_ppc} is_ppc_redirect={is_ppc_redirect} />
+            FooterNav = <LoadableFooter is_ppc={is_ppc} is_ppc_redirect={is_ppc_redirect} />
             break
         case 'besquare':
             Navigation = <NavCareers is_besquare />
             FooterNav = <BeSquareFooter />
             break
         case 'careers':
             Navigation = <NavCareers />
-            FooterNav = <Footer no_language={true} type={type} />
+            FooterNav = <LoadableFooter no_language={true} type={type} />
             break
         case 'transparent':
+            Navigation = <NavTransparent is_ppc_redirect={is_ppc_redirect} is_ppc={is_ppc} />
```

Comment:
Hi @NikitK-deriv I'm just curious to know what is the `is_ppc` exactly? 🙏 

Also may I know how we are fixing the issue exactly I couldn’t understand it, many thanks in advance 🙏

Reply:
Yes, sure @mitra-deriv 

Seems like `is_ppc` property related to landing pages. For some special cases. I'll check landing for payment methods and maybe have sense to remove that. Good catch 👍 

So main issue was that footer component didn't exist after reload page due that footer for payment methods used `Loadable` function to load component later. That's why footer flash every time
Common behaviour of react is show whole page when all components already rendered. I decided to import component without loadable function and save common react behavior for this specific page

Hope it's clear 🙇

Reply:
Hi @NikitK-deriv , as the issue is for Payment Method page, can you implent this only for that page? If needed we can implement for all pages for a different card

Reply:
Hey @habib-deriv it's exactly implemented only for payment agent page. The other didn't change